go/cmd/vttablet/vttablet.go

```diff
@@ -43,12 +43,17 @@ func main() {
 	dbconfigs.RegisterFlags(dbconfigFlags)
 	mysqlctl.RegisterFlags()
 	flag.Parse()
-	tabletenv.Init()
 	if len(flag.Args()) > 0 {
 		flag.Usage()
 		log.Errorf("vttablet doesn't take any positional arguments")
 		exit.Return(1)
 	}
+	if err := tabletenv.VerifyConfig(); err != nil {
+		log.Errorf("invalid config: %v", err)
+		exit.Return(1)
```

Comment:
Changing to log.Fatalf sounds fine.

Ideally, I would like to call `VerifyConfig` from `NewServer()` and let it return an error if there's a problem. But I was concerned this would draw in more code changes :)